src/main/kotlin/com/amazon/ion/plugin/intellij/Ion.bnf

```diff
@@ -0,0 +1,45 @@
+{
+  parserClass="com.amazon.ion.plugin.intellij.IonParser"
+  extends="com.intellij.extapi.psi.ASTWrapperPsiElement"
+  psiClassPrefix="Ion"
+  psiImplClassSuffix="Impl"
+
+  psiPackage="com.amazon.ion.plugin.intellij.psi"
+  psiImplPackage="com.amazon.ion.plugin.intellij.psi.impl"
+
+  elementTypeHolderClass="com.amazon.ion.plugin.intellij.psi.IonTypes"
+  elementTypeClass="com.amazon.ion.plugin.intellij.psi.IonElementType"
+  tokenTypeClass="com.amazon.ion.plugin.intellij.psi.IonTokenType"
+  psiImplUtilClass="com.amazon.ion.plugin.intellij.psi.impl.IonPsiUtil"
+}
+
+ionFile ::= item_*
+private item_ ::=  value
+
+object ::= (struct | array | expression)
+struct ::= LBRACE [members] RBRACE {pin=1}
+members ::= struct_pair*
+pair ::= COMMENT* (key SEPARATOR value)
+key ::= (QUOTE KEY_NAME QUOTE) | (QQUOTE KEY_NAME QQUOTE) | (QQQUOTE KEY_NAME QQQUOTE) | KEY_NAME {methods=[getKeyName]}
+array ::= LBRACKET [elements] RBRACKET {pin=1}
+elements ::= array_element*
+expression ::= (LPAREN [expression_operator] [atoms] RPAREN)
+expression_operator ::= (annotation)* (symbol)
```

Comment:
I can change the name to `sexp`. The reason why I am picking up `expression_operator` is because there are features within the Ion plugin which would be useful to parse the first operator seperately. In my mind the BNF grammar is a combination of the language spec and useful "tags" that would make language features easier to implement.